### PR TITLE
Correct commands to run WSJ experiments

### PR DESCRIPTION
**Pull request by nke001**

### CODE REVIEW

File: exp/wsj/write_hdf_dataset.sh [hunk 1]
```diff
@@ -94,11 +94,11 @@ $LVSR/bin/kaldi2fuel.py $h5f add_text --applymap tmp_chars.txt tmp_chars_all cha
 compute-fbank-feats --use-energy=true --num-mel-bins=40 "scp:tmp_convert_all_wav.scp" 	"ark:-" | \
 	add-deltas ark:- ark,scp:tmp_fbank40.ark,tmp_fbank40.scp
 
-compute-global-cmvn-stats.py \
+scripts/compute-global-cmvn-stats.py \
```
Comment (dmitriy-serdyuk) [inline exp/wsj/write_hdf_dataset.sh]: Actually, it should be without `scripts`. We install this script [here](https://github.com/dmitriy-serdyuk/kaldi-python/blob/master/setup.py#L20).

**Comment by dmitriy-serdyuk** [comment]
Thanks!

I remember you found one more place where we didn't mention the correct path. Do you remember where?

File: exp/wsj/README.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ Check that `$LVSR` environment variable points to this repository and
    model option, beam size and subset there are alignment images and 
    `report.txt` which contains transcripts, approximate CER and WER and other 
    auxiliary information.
+   <part> specifies the dataset to decode. For example it can take on the value 'valid'
```
Comment (dmitriy-serdyuk) [inline exp/wsj/README.md]: Or 'test', it should be more useful to people who want to repeat the experiment. + full stop.

**Comment by rizar** [comment]
I guess it is outdated, please let me know if I am wrong.

**Pull request closed without merging**